### DISCUSSION TITLE
Question about  pre-trained backbone network

### DISCUSSION THREAD

**Discussion by zxb-caffe** [STVIR/pysot#272] [General]
Hello, 
can you provide Baidu pan links to the pre-trained backbone network?
能不能提供预训练骨干网的百度网盘链接？

**Comment by zy5037** [comment]
> Hello,
> can you provide Baidu pan links to the pre-trained backbone network?
> 能不能提供预训练骨干网的百度网盘链接？

你好,骨干网络是因为源代码没提供训练方法所以训练不了吗,我每当开始训练骨干网络的时候代码就会报错说backbone没parameters这个属性.
